docs/UserGuide.md

```diff
@@ -125,16 +135,18 @@ If a command was not successfully executed, the text within the command input bo
 </div>
 
 ### 2. Buyer List
-You can find the buyer list located at the left section of Cobb's GUI.
-![Buyer List](images/BuyerList.png)<br>
-The __Buyer List__ displays information regarding buyers who are currently stored in Cobb's database.
+You can find the buyer list located at the left section of the window.<br>
+![Buyer List](images/BuyerList.png)<br><br>
+The __Buyer List__ displays information regarding buyers who are currently stored in Cobb's database. The buyer list
+might not be showing **all* the buyers in the database all the time (see [FAQ](#faq)).
 
 The buyer list can be filtered and modified using commands given in the [features](#features) section of the guide.
 
 ### 3. Property List
 You can find the property list located at the right section of Cobb's GUI.
```

Comment:
```suggestion
You can find the property list located at the right section of the window.
```
For consistency!

Reply:
Removed all mentions of GUI in the UG as well